appyters/PrismEXP/PrismEXP.ipynb

```diff
@@ -0,0 +1,522 @@
+    "                section='GMT'\n",
+    "            )\n",
+    "        ],\n",
+    "        'Upload': [\n",
+    "            FileField(\n",
+    "                name='gmt_upload',\n",
+    "                label='Upload GMT file',\n",
+    "                description='Upload a GMT file. Each line represents a gene function and the associated genes separated by TAB.',\n",
+    "                examples={\n",
+    "                    'KEGG_2019_Mouse.gmt': \"https://mssm-prismx.s3.amazonaws.com/misc/KEGG_2019_mouse.gmt\"\n",
+    "                },\n",
+    "                section='GMT'\n",
+    "            ),\n",
+    "        ],\n",
+    "    },\n",
+    "    section='GMT'\n",
+    ") %}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter markdown\n",
+    "\n",
+    "<h1> <img align=\"left\" width=200 src=\"https://mssm-prismx.s3.amazonaws.com/images/prismxsmall.png\">\n",
```

Comment:
You might want to add `class="img-fluid"` to play well with bootstrap, typically I do this automagically when I render markdown.

Reply:
this part behaved very strangely and would result in completely wrong formatting just by adding white spaces at the wrong place.